applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -133,7 +133,6 @@ static bool adv_data_parse(struct bt_data *data, void *user_data)
 	if (data->type == BT_DATA_BROADCAST_NAME && data->data_len) {
```

Comment:
Now adv_data_parse() expects an array where all elements are '\0'?
If this is true we should at least make a comment/note about it